pkg/util/logging_http_transport.go

```diff
@@ -56,11 +55,7 @@ func (t *LoggingHttpTransport) RoundTrip(r *http.Request) (*http.Response, error
 	for k, v := range r.Header {
 		if sensitiveRequestHeaders.Has(k) {
 			redacted[k] = v
-			l := 0
-			for _, h := range v {
-				l += len(h)
-			}
-			r.Header.Set(k, strings.Repeat("*", l))
+			r.Header.Set(k, "(REDACTED FOR PRIVACY)")
```

Comment:
Do we really have to be that verbose? I would be happy to just show 8 asterisks ("**********"). This seems to be a common practice when it comes to erasing passwords. 

Why for 'privacy' ? And not for 'security' ? Is it because the password reveals something personal ?. And "redacted" means to be changed based on the value, but it's just "overwritten" without condition, not "redacted".

Nit-picking on high-level, sorry ;-) Just saying, that I really would prefer a simple commonly used, non-sense value like "**********", as, because it's non-sense, nobody questions its sense ;-)

If my comment goes over the top, feel free to merge nevertheless.

Reply:
I think we just need to agree on the verbose to show instead of the real authorization string.

Reply:
Let's do a quick vote.

Reply:
Option 1:`(REDACTED FOR PRIVACY)`

Reply:
Option 2: `********`

Reply:
In my first version of this PR it was "ELIDED", and people were like "what does ELIDED even mean?" so I think asterisks are fine.

Reply:
WG meeting approves the asterisks.

Reply:
Will update today. Thank you all.